lms/static/styles/frontend_apps.scss

```diff
@@ -14,27 +8,28 @@
 @use 'components/heading';
 @use 'components/label';
 
-// Generic React components.
-// Note: Some of these are also used by server-rendered HTML templates, not
-// just by React code.
+// Components rendered both by Preact and other HTML (eg. server-rendered
+// templates).
 @use 'components/Button';
-@use 'components/Dialog';
-@use 'components/Table';
 
-// React components.
+// Preact components.
```

Comment:
The period (`.`) is a little odd here (again, trivial nit)

Reply:
Yeah. I've removed them. It doesn't seem appropriate for headings. I think it would be useful to document specific advice about this in our frontend style guide, even if it seems obvious.